### ISSUE TITLE
keyboard focus isn't in category list

### ISSUE THREAD

**Issue by elementaryBot** [elementary/appcenter#38]
After choosing a category, keyboard focus should go into the list. You should be able to begin moving through the list with the keyboard without first focusing with tab

Launchpad Details: [#LP1525684](https://bugs.launchpad.net/bugs/1525684) Daniel Fore - 2015-12-13 18:52:06 +0000

**Comment by danrabbit** [comment]
I guess this is fixed by #1270
